HttpKernel/ControllerInjectorsWarmer.php

```diff
@@ -28,12 +28,21 @@ class ControllerInjectorsWarmer implements CacheWarmerInterface
     private $kernel;
     private $controllerResolver;
     private $blackListedControllerFiles;
+    private $scanAllBundles;
+    private $scanBundles;
 
-    public function __construct(KernelInterface $kernel, ControllerResolver $resolver, array $blackListedControllerFiles)
-    {
+    public function __construct(
+        KernelInterface $kernel,
+        ControllerResolver $resolver,
+        array $blackListedControllerFiles,
+        $scanAllBundles,
+        array $scanBundles
```

Comment:
Should be optional and default to `array()`. That way, the constructor is still callable like it was before.

Reply:
I'm going to push the changes. Out of curiosity, why provinding default values just to make the constructor callable like before? Service is created and managed by Symfony DI, so why bother?

Reply:
Because who knows who might use it without sf DI?